src/main/kotlin/com/fantasy/ladbe/controller/PlayerController.kt

```diff
@@ -0,0 +1,40 @@
+package com.fantasy.ladbe.controller
+
+import com.fantasy.ladbe.common.dto.PageDto
+import com.fantasy.ladbe.common.web.CommonApiResponse
+import com.fantasy.ladbe.common.web.CommonApiResponse.Companion.success
+import com.fantasy.ladbe.dto.PlayerDto
+import com.fantasy.ladbe.service.PlayerService
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("api/players")
+class PlayerController(
+    val playerService: PlayerService,
+) {
+
+    @GetMapping
+    fun getPlayer(@RequestParam id: Long): ResponseEntity<CommonApiResponse> {
+        val body = playerService.readOne(id)?.let { success(it) }
+        return ResponseEntity(body, HttpStatus.OK)
+    }
+
+    @GetMapping("/all")
+    fun getAllPlayers(): ResponseEntity<CommonApiResponse> {
+        val body = playerService.readAll()?.let { success(it) }
+        return ResponseEntity(body, HttpStatus.OK)
+    }
```

Comment:
특별한 경우가 아니면 메소드의 return value 는 nullable 하지 않게 작성하는게 좋아요. 일단 `playerSerice.readAll()` 은 데이터가 없으면 null 이 아닌 empty list 를 주면 되구요, 그 메소드 return type 을 nun-null 로 바꾸면 아래와 같이 표현 가능해요. (function 의 return type 을 꼭 명시하지 않아도 됨)

```suggestion
    fun getAllPlayers() =
        ResponseEntity(success(playerService.readAll()), HttpStatus.OK)
```